13-multiplecomponents-lab/src/01-redspotter.jsx

```diff
@@ -1,13 +1,79 @@
 /**
- * Lab 01:
+ * Lab 02:
  *
- * Write a game that shows 10 squares: 9 gray and 1 red.
- * When a user clicks on the red she gets 5 points and red changes its place.
- * When a user clicks on any other square nothing happens.
+ * Implement a number guessing game component.
  *
+ * The component should provide an input field for the user to guess a
+ * number picked randomly by the computer (from 1 to 1,000).
+ * When typing a number, the component should print "too low" or "too high"
+ * as a hint to the user.
+ *
+ * To make the game interesting, computer should sometimes cheat and print wrong messages.
  */
 
+/**
+ * Created by ofek_am on 18/12/2016.
+ */
 import React from 'react';
+import _ from 'underscore';
+
+export default class RedSpotter extends React.Component{
+
+    static propTypes = {
+
+    };
+
+    static defaultProps = {
+
+    };
+
+    boxes=[
```

Comment:
instead of hardcoding boxes classNames inside an array consider setting one variable in state: red box index. also add class property numberOfBoxes. In such a way you won't need to check classNames when clicking box